pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -576,18 +576,21 @@ func (m *managerImpl) evictPod(pod *v1.Pod, resourceName v1.ResourceName, evictM
 		glog.Errorf("eviction manager: cannot evict a critical pod %s", format.Pod(pod))
 		return false
 	}
+	msg := fmt.Sprintf(message, resourceName)
 	status := v1.PodStatus{
 		Phase:   v1.PodFailed,
-		Message: fmt.Sprintf(message, resourceName),
+		Message: msg,
 		Reason:  reason,
 	}
 	// record that we are evicting the pod
 	m.recorder.Eventf(pod, v1.EventTypeWarning, reason, evictMsg)
 	gracePeriod := int64(0)
 	err := m.killPodFunc(pod, status, &gracePeriod)
 	if err != nil {
+		m.recorder.Eventf(pod, v1.EventTypeWarning, "KubeletEviction", "Error deleting: %v", err)
 		glog.Errorf("eviction manager: pod %s failed to evict %v", format.Pod(pod), err)
 	} else {
+		m.recorder.Eventf(pod, v1.EventTypeNormal, "KubeletEviction", "Successfully deleted Pod as %s", msg)
```

Comment:
@gmarek - do we consider eviction something that could be noticed from resource alteration ?

Reply:
Yup - it's basically a time when Pod enters 'Terminated' (or something like that) phase.